### PR TITLE
[filter/tflite] Add enum for tflite delegates

### PR DESCRIPTION
**Pull request by anyj0527**
- Add enum for delegates
- Refactor parsing custom props

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by jaeyun-jung — APPROVED** [review]

**Review by again4you — APPROVED** [review]
LGTM!

File: ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc [hunk 1]
```diff
@@ -66,12 +66,24 @@
...
 #endif
 
+/**
+ * @brief Possible tensorflow-lite delegates.
+ */
+typedef enum {
+  TFLITE_DELEGATE_NONE = 0,
+  TFLITE_DELEGATE_GPU,
+  TFLITE_DELEGATE_NNAPI,
+
```
Comment (helloahn) [inline ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc:76]: Q) Is there any intention for this empty line?

Comment (anyj0527) [reply ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc:76]: Oh it's for the new delegates, XNNPACK

**Review by myungjoo — COMMENTED** [review]
LGTM

**Pull request merged**